modules/light-clients/08-wasm/client/cli/tx.go

```diff
@@ -82,3 +82,41 @@ func newSubmitStoreCodeProposalCmd() *cobra.Command {
 
 	return cmd
 }
+
+func newMigrateContractCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "migrate-contract [client-id] [checksum] [migrate-msg]",
```

Comment:
I guess most contracts for now will not do anything with this message, so most of the times it will be just `{}`? I am fine supporting it already now or adding support for it later.